### PR TITLE
Added Earthly to list

### PR DESCRIPTION
**Pull request by biplobsd**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by crweiner** [comment]
Hello @biplobsd! 👋 Thanks for your PR! Can you please also add Earthly to the A->Z section of the List? I'd love to get Earthly on the list soon, so I'd appreciate if you could take care of that soon! The two sections are formatted slightly differently, so be sure to follow the ```Contributing.md``` file. 👍

**Comment by biplobsd** [comment]
> Hello @biplobsd! 👋 Thanks for your PR! Can you please also add Earthly to the A->Z section of the List? I'd love to get Earthly on the list soon, so I'd appreciate if you could take care of that soon! The two sections are formatted slightly differently, so be sure to follow the `Contributing.md` file. 👍

Thank you @crweiner for saying that. I am enjoying to update this to A->Z section. Please look again my PR. Is it now all right now?

File: README.md [hunk 1]
```diff
@@ -145,6 +145,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
 
+### E
+
+#### **Earthly**
+
+- **Swag**: Stickers (limit 40)
+- **Requirements**: Contribute code to the [Earthly](https://github.com/earthly/earthly) open source build tool or contribute an Earthfile build to any public repo.
+- **How to sign up**: [Grab an issue](https://github.com/earthly/earthly/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) and complete your PR after accepted please fill out this [form](https://forms.gle/DVkrWCu4yD2wMPmE8).
+- **Notes**: If you need any assistance, [join our slack channel](https://earthly.dev/slack). We are here to help! See our [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
+
```
Comment (crweiner) [inline README.md:156]: Can you move this into proper alphabetical order @biplobsd? 😄 Let's place "E" after "D", please 😆,

**Review by crweiner — CHANGES_REQUESTED** [review]
Please place it in the proper alphabetical spot! 🤣

File: README.md (hunk 1 above)
Comment (biplobsd) [reply README.md]: 😃 Sorry for your time!

**Comment by crweiner** [comment]
Thank you for making that change and for adding Earthly to the list! I hope we can send some great PRs their way 🎉 🎉 

Have a wonderful Hacktoberfest and thanks for making the Swag List even better! 

thank you

File: README.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Swag**: Stickers (limit 40)
 - **Requirements**: Contribute code to the [Earthly](https://github.com/earthly/earthly) open source build tool or contribute an Earthfile build to any public repo.
 - **How to sign up**: [Grab an issue](https://github.com/earthly/earthly/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) and complete your PR after accepted please fill out this [form](https://forms.gle/DVkrWCu4yD2wMPmE8).
-- **Notes**: If you need any assistance, [join our slack channel](https://earthly.dev/slack). We are here to help! See our [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
+- **Notes**: If you need any assistance, [join thjeir Slack channel](https://earthly.dev/slack). See their [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
```
Comment (biplobsd) [inline README.md:115]: thjeir Slack -> their Slack

Comment (crweiner) [reply README.md:115]: I swear I can type.... 😆 Fixing that now

**Pull request merged**